tests/NLog.UnitTests/Config/LogFactorySetupTests.cs

```diff
@@ -568,5 +569,103 @@ public void SetupBuilderLoadConfigurationFromXmlPatchTest()
             Assert.False(logFactory.AutoShutdown);
             Assert.Single(logFactory.Configuration.Variables);
         }
+
```

Comment:
Maybe include a test-method where you call LogFactory.Close() ? (Similar to Shutdown).